debian/rules

```diff
@@ -13,9 +13,9 @@ endif
 
 DEBVERS := $(shell dpkg-parsechangelog | sed -n -e 's/^Version: //p')
 VERSION := $(shell echo '$(DEBVERS)' | sed -e 's/^[\[:digit:]]*://' -e 's/[-].*//')
-MAJOR := $(shell echo '$(VERSION)' | awk -F . {'print $$1'})
-MINOR := $(shell echo '$(VERSION)' | awk -F . {'print $$2'})
-REV := $(shell echo '$(VERSION)' | awk -F . {'print $$3'})
+MAJOR := $(shell echo '$(VERSION)' | awk -F '[.+]' {'print $$1'})
+MINOR := $(shell echo '$(VERSION)' | awk -F '[.+]' {'print $$2'})
+REV := $(shell echo '$(VERSION)' | awk -F '[.+]' {'print $$3'})
```

Comment:
The Rev Value generated by the Makefile etc is already "Commits" since tag... 
so 1.4.1 is 1 commit after 1.4 was tagged on master branch.